pkgs/development/python-modules/openstep-parser/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "openstep-parser";
+  version = "2.0.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "kronenthaler";
+    repo = "openstep-parser";
+    rev = "refs/tags/2.0.0";
+    hash = "sha256-xBxbSKiF6zVTwZGGjPADx1LkhQjX2VHZsct/xf3rkTs=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
```

```suggestion
  build-system = [ setuptools ];
```
see https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters